.github/ISSUE_TEMPLATES/feature_request.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for the connector
+title: ''
+labels: feature-request
+assignees: ''
+
+---
```

Comment:
The file needs a `.md` extension, same for the other template.

And the overall directory should be singular, i.e. `ISSUE_TEMPLATE`.

Reply:
I've already fixed this.